hack/install.sh

```diff
@@ -55,16 +60,16 @@ do_install() {
 
 	if command_exists docker; then
 		cat >&2 <<-'EOF'
-			Warning: the "docker" command appears to already exist on this system.
+		Warning: the "docker" command appears to already exist on this system.
 
-			If you already have Docker installed, this script can cause trouble, which is
-			why we're displaying this warning and provide the opportunity to cancel the
-			installation.
+		If you already have Docker installed, this script can cause trouble, which is
+		why we're displaying this warning and provide the opportunity to cancel the
+		installation.
 
-			If you installed the current Docker package using this script and are using it
-			again to update Docker, you can safely ignore this message.
+		If you installed the current Docker package using this script and are using it
+		again to update Docker, you can safely ignore this message.
 
-			You may press Ctrl+C now to abort this script.
+		You may press Ctrl+C now to abort this script.
```

Comment:
Just making sure all this whitespace changing is intentional.  This one has no effect on the output (unlike the previous one I commented on), so I'm indifferent unless it was unintentional.